lib/sts/mapper.rb

```diff
@@ -0,0 +1,436 @@
+
+require "shale"
```

Comment:
@HassanAkbar why is there a different implementation of Mapper?

Reply:
@ronaldtse As mentioned in the [shale documentation](https://github.com/kgiszczak/shale#mapping-xml-elements-and-attributes-to-object-attributes)

```asciidoc
`map_content` - map first text node to attribute
```

`Shale::Mapper` only uses the first text element inside the tag as content and discard every other text element and it also changes the order of the tags. 

I have overridden  the `of_xml` and `as_xml` methods of `Shale::Mapper` to respect the order of the tags and read all the content.

Reply:
Agree with the enhancement. However, I think we should probably collect these enhancements to Shale in another location, and instead of a mix-and-match of using Shale class vs this overridden class, we should only use the overridden class.

In fact, I want to enhance our LutaML gem to support Ruby object access such as Shale does. LutaML already provides a way to define a data model, we just need to extend it to support XML parsing and Shale-type mapping.

The conclusion is I'd like to update the LutaML gem to contain this Shale hack... @HassanAkbar can you help plan this feature at LutaML?

Reply:
>  @HassanAkbar can you help plan this feature at LutaML?

@ronaldtse Sure, I will prioritize this in this ticket in LutaML. ->  https://github.com/lutaml/expressir/issues/144

Reply:
Moving this ticket to a separate issue, merging this first.